src/Symfony/Installer/NewCommand.php

```diff
@@ -415,13 +415,19 @@ protected function generateComposerProjectName()
      */
     private function fixComposerPackageName($name)
     {
-        return strtolower(
-            preg_replace(
-                array('/([A-Z]+)([A-Z][a-z])/', '/([a-z\d])([A-Z])/'),
-                array('\\1-\\2', '\\1-\\2'),
-                strtr($name, '-', '.')
-            )
+        $name = preg_replace(
+            array('/([A-Z]+)([A-Z][a-z])/u', '/([a-z\d])([A-Z])/u'),
+            array('\\1-\\2', '\\1-\\2'),
+            strtr($name, '-', '.')
         );
+
+        if (function_exists('mb_strtolower')) {
+            $name = mb_strtolower($name);
+        } else {
+            $name = utf8_encode(strtolower(utf8_decode($name)));
```

Comment:
accented chars are not valid in composer package names anyway: https://github.com/composer/composer/blob/d7d91269ec4fb0da171a1c38fa1a47c493627dc0/src/Composer/Package/Loader/ValidatingArrayLoader.php#L52